src/lsp-client/clientTypes.ts

```diff
@@ -119,7 +119,7 @@ export interface CoqLspServerConfig {
     client_version: string;
     eager_diagnostics: boolean;
     goal_after_tactic: boolean;
-    show_coq_info_messages: boolean;
+    show_waterproof_info_messages: boolean;
```

Comment:
```suggestion
    show_coq_info_messages: boolean;
```

We can't rename these as they are passed to the `coq-lsp` instance and parsed there. So internally we have to keep the 'coq' name, while for the external/user facing side 'waterproof' is fine.